src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -448,14 +448,13 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                         if (intrin.op3->isContained())
                         {
                             assert(intrin.op3->IsVectorZero());
-                            if (intrin.op1->isContained())
+                            if (intrin.op1->isContained() || intrin.op1->IsMaskAllBitsSet())
```

Comment:
The `intrin.op1->isContained()` seems pointless here, no?

We just need to know its not contained for an instruction that actually requires its use (like `abs`)

Reply:
i will have to double check on this one. There was lot of back and forth on this for certain scenarios. I will verify it on next round.